### ISSUE TITLE
Clean up job folders after usage is complete

### ISSUE THREAD

**Issue by WardBrian** [flatironinstitute/stan-playground#187]
We want to keep the compiled targets for caching, but the actual job folder is still a one-and-done

**Comment by magland** [comment]
> We want to keep the compiled targets for caching, but the actual job folder is still a one-and-done

Perhaps it should be this way. But right now the main.js and wasm ends up on the job folder. But even in that case, it makes sense to delete them after a time.

**Comment by WardBrian** [comment]
Yes, right now we assume the url will be in the job folder in the frontend:

https://github.com/flatironinstitute/stan-playground/blob/49bfb9451db76579756865325eda6e4d201d027e/gui/src/app/compileStanProgram/compileStanProgram.ts#L57

It would be relatively easy to have the backend instead reply with a link, which we could point straight into the cache, rendering the job folder irrelevant after that
